frontend/src/views/admin/offertable/index.tsx

```diff
@@ -132,11 +134,12 @@ export function ConnectedOfferTable(props) {
     // are generated on-the-fly, memoize the result so we don't trigger unneeded re-renders.
     const columns = React.useMemo(() => {
         // Bind an `ApplicantCell` to a particular field
-        function generateApplicantCell(field) {
-            return (props) => (
+        function generateApplicantCell(field: any) {
+            return (props: any) => (
                 <ApplicantCell
                     field={field}
-                    upsertApplicant={(...args) =>
+                    upsertApplicant={(...args: any) =>
+                        // @ts-ignore
```

Comment:
I've thought typescipt was giving an error for no reason many times. Almost always, I was wrong...

Any valid use of `// @ts-ignore` should be accompanied by documentation about why that's really the correct solution. In this case, I don't think it's right.

Reply:
This part I need some further investigation. The code works as-is and breaks when I remove any of the `...` (which makes sense, `...` makes many arguments, without `...` there is only one argument); therefore it is confusing why ts is complaining about this. It might be some underlying babysitting done by javascript to magically make incorrect code work as intended.